### PR TITLE
Mae option

### PR DESCRIPTION
**Pull request by a1xt06**
Added a mae option (false by default)

### CODE REVIEW

File: ngboost_tuner/tune.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import pickle
 from pathlib import Path
 import os
+from math import sqrt
```
Comment (ryan-wolbeck) [inline ngboost_tuner/tune.py]: doesn't seem to be used

File: ngboost_tuner/cli.py [hunk 2]
```diff
@@ -150,4 +150,12 @@ def build_cli():
         help="Set to true for lightgbm as base regresor. Default $LIGHTGBM",
     )
```
Comment (ryan-wolbeck) [inline ngboost_tuner/cli.py]: remove the space/apply black

File: ngboost_tuner/tune.py [hunk 3]
```diff
@@ -171,7 +179,7 @@ def objective(params):
                 early_stopping_rounds=2,
             )
 
-            loss = ngb.evals_result["val"]["LOGSCORE"][ngb.best_val_loss_itr]
+            loss = ngb.evals_result["val"][score.upper()][ngb.best_val_loss_itr]
```
Comment (ryan-wolbeck) [inline ngboost_tuner/tune.py]: errors here..
`AttributeError: type object 'CRPScore' has no attribute 'upper'`

File: ngboost_tuner/cli.py [hunk 4]
```diff
@@ -149,5 +149,12 @@ def build_cli():
         default=bool(os.getenv("LIGHTGBM")),
         help="Set to true for lightgbm as base regresor. Default $LIGHTGBM",
     )
+    tune.add_argument(
+        "-mae",
+        "--mae_loss",
+        type=bool,
+        default=False,
+        help="Set to true for median absolute error as loss function. Default False",
```
Comment (ryan-wolbeck) [inline ngboost_tuner/cli.py]: this is mean right?

Comment (a1xt06) [reply ngboost_tuner/cli.py]: yes it is...dang will change

**Pull request merged**